make/release

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
+SRC=`basename $(pwd)`
```

Comment:
Not sure if its an overkill but there are ["safer" ways to get the script's dir](https://stackoverflow.com/questions/59895/getting-the-source-directory-of-a-bash-script-from-within). It it is working maybe we can get away with this.